### ISSUE TITLE
C# 10 File Scoped Namespaces break Syntax Highlighting

### ISSUE THREAD

**Issue by gustavo-shigueo** [microsoft/vscode#136195]
I'm on Windows 10, VSCode version 1.61.2, using .NET 6.0 with Windows Forms. Attempting to use file scoped namespaces is causing issues to syntax highlighting. I have not observed any errors with intellisense or sytax error detection, only highlighting

Regular behavior:
![image](https://user-images.githubusercontent.com/58121396/139595738-f2b0cd68-314f-4a2d-ac00-0a3cb94b0dc4.png)

When using file scoped namespaces:
![image](https://user-images.githubusercontent.com/58121396/139595783-d3ae86f5-1c20-4205-9326-a2472543afe9.png)

**Comment by alexr00** [comment]
@gustavo-shigueo can you please provide a copy/pastable code snippet that I can use to reproduce the issue?

**Comment by yume-chan** [comment]
dotnet/csharp-tmLanguage#225

**Comment by gustavo-shigueo** [comment]
```cs
namespace WindowsFormsApp1;

static class Program
{
    /// <summary>
    ///  The main entry point for the application.
    /// </summary>
    [STAThread]
    static void Main()
    {
        ApplicationConfiguration.Initialize();
        Application.Run(new Form1());
    }
}
```

**Comment by alexr00** [comment]
@gustavo-shigueo thank you for the repro snippet.

Thanks for the link to the upstream issue @yume-chan! This is indeed the correct upstream issue. I am pulling the in the latest from the upstream repo today and the fix will be in the first 1.63 insiders build.
